subiquity/tests/api/test_api.py

```diff
@@ -386,7 +386,7 @@ async def test_v2_flow(self):
         cfg = self.machineConfig("examples/machines/simple.json")
         with cfg.edit() as data:
             attrs = data["storage"]["blockdev"]["/dev/sda"]["attrs"]
-            attrs["size"] = str(25 << 30)
+            attrs["size"] = str(10 << 30)
```

Comment:
Is that not the size in the json already?

Reply:
It used to be, we added a zero to the default size of this file since it's so high traffic for dry-run testing and allows for more stuff.
Also I like the idea of at least one API test having a low number like this, so that it's independent of whatever other tweaks simple.json might see.